### ISSUE TITLE
Support for Chiaki navigation using a controller

### ISSUE THREAD

**Issue by veerpandya** [thestr4ng3r/chiaki#414]
**What platform does your feature request apply to?**
Desktop.  My use case is specifically for Raspberry Pi OS, but it would be convenient on any desktop os.

**Is your feature request related to a problem? Please describe.**
I want to integrate Chiaki into my RetroPie setup.  This means I would ideally not have a keyboard or mouse connected and just have a controller for navigation.  Currently I need to use a mouse to select the PS4 and begin the remote play session.

**Describe the solution you'd like**
It would be incredibly convenient to be able to use a connected controller to navigate the Chiaki interface and begin the remote play session.  Using the dpad of the controller to highlight options and then having X or O to select the option would be great.

**Describe alternatives you've considered**
The current alternative I have is to use a program like qjoypad to map the controller to mouse movement and clicks and then navigate the menu that way.
 
**Additional context**
In summary, it would be really helpful if Chiaki menu navigation could be done using a connected controller instead of a mouse.
